alacritty/src/window.rs

```diff
@@ -1,52 +1,46 @@
-use std::convert::From;
+#[rustfmt::skip]
```

Comment:
This is required to preserve the order of elements and white space between them.